CHANGELOG.md

```diff
@@ -481,6 +490,10 @@ Issue [#33879](https://github.com/grafana/grafana/issues/33879)
 - **Dashboard:** Remove support for loading and deleting dashboard by slug. [#35104](https://github.com/grafana/grafana/pull/35104), [@dsotirakis](https://github.com/dsotirakis)
 - **InfluxDB:** Deprecate direct browser access in data source. [#35105](https://github.com/grafana/grafana/pull/35105), [@gabor](https://github.com/gabor)
 - **VizLegend:** Add a read-only property. [#35096](https://github.com/grafana/grafana/pull/35096), [@dprokop](https://github.com/dprokop)
+- **Runtime reload**: Introduce runtime reload for SAML settings (Enterprise)
+- **FGAC**: Introduce fine grained access control (Enterprise)
```

Comment:
@sakjur what should this say, I'm thinking you might not want it to say FGAC :)

Reply:
```suggestion
- **Access control**: Introduce fine grained access control beta (Enterprise)
```